CsvParser/csvParser_test.go

```diff
@@ -94,7 +94,7 @@ func Test_CsvParserFourthEntry(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			incidents := CsvParser(tt.input)
+			incidents := CsvParser(tt.input, 51)
```

Comment:
Please add a comment explaining why this "magic number" is here.

Reply:
Removed the magic number